catalog-rest-service/src/main/resources/json/data/policy/DataStewardAccessControl.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "DataStewardRoleAccessControlPolicy",
   "displayName": "Data Steward Role Access Control Policy",
+  "fullyQualifiedName": "DataStewardRoleAccessControlPolicy",
```

Comment:
@Sachin-chaurasiya  This is the trouble maker on why policy is missing.

The policy needs an fqn so that relationships can be established